core/store/src/flat/types.rs

```diff
@@ -3,13 +3,19 @@ use near_primitives::errors::StorageError;
 use near_primitives::hash::CryptoHash;
 use near_primitives::types::BlockHeight;
 
-#[derive(BorshSerialize, BorshDeserialize, Clone, PartialEq, Eq)]
+#[derive(BorshSerialize, BorshDeserialize, Debug, Clone, PartialEq, Eq)]
 pub struct BlockInfo {
     pub hash: CryptoHash,
     pub height: BlockHeight,
     pub prev_hash: CryptoHash,
 }
 
+impl BlockInfo {
+    pub fn genesis(hash: CryptoHash) -> BlockInfo {
+        BlockInfo { hash, height: 0, prev_hash: CryptoHash::default() }
```

Comment:
This is not exactly the case, genesis height is configurable and is set somewhere in `ChainGenesis`. For mainnet it is 9820210.
Maybe we don't care about exact genesis height in `BlockInfo`, as 0 is strictly smaller than anything else, but we need to justify and comment that in such case.

Reply:
ah, indeed, thanks for pointing that out. I've added parameter for genesis height: ebd7827d2